### DISCUSSION TITLE
Q: Automatic closing of "Send" window possible?

### DISCUSSION THREAD

**Discussion by tmx7777** [dannagle/PacketSender#244] [Q&A]
Hi, first of all, this is a great tool that is helping me a lot.

I was wondering though, if it would be possible to automatically close the "Send" window. I have the following scenario: I use PacketSender as a TCP server to test a TCP client implementation of mine. After the connection has been established, a window opens (which I meant by "Send" window) where I can send data back to the client. It would be really helpful, if this window could be closed automatically or at least be re-used after closing the connection from the client side. I'm doing a lot of (semi-)automatic testing where I open a lot of connections, send some data and then close the connection. So over time, a huge number of "Send" windows are accumulating.

Thanks for any help.

**Comment by dannagle** [comment]
You can turn off persistent connections to suppress that window. 

Uncheck this box:

![image](https://user-images.githubusercontent.com/1140573/124755425-f93d6600-def0-11eb-8349-27d54913e9c3.png)

Is that what you are looking for?
